Source/WebCore/page/RemoteFrame.h

```diff
@@ -74,6 +77,8 @@ class RemoteFrame final : public Frame {
     RefPtr<RemoteFrameView> m_view;
     UniqueRef<RemoteFrameClient> m_client;
     Markable<LayerHostingContextIdentifier> m_layerHostingContextIdentifier;
+    std::optional<FrameIdentifier> m_remoteFrameID;
```

Comment:
This should always be the same as this frame's frame identifier, so just remove this.